Framework/API/src/MatrixWorkspace.cpp

```diff
@@ -72,11 +72,6 @@ MatrixWorkspace::MatrixWorkspace(const MatrixWorkspace &other)
   m_isCommonBinsFlag = other.m_isCommonBinsFlag;
   m_masks = other.m_masks;
   // TODO: Do we need to init m_monitorWorkspace?
-
```

Comment:
Am I correct in saying that this is no longer required because of the new copy behaviour in `setInstrument`?